cobc/ChangeLog

```diff
@@ -97,8 +97,8 @@
 	* typeck.c (cb_build_expr): remove cb_ prefix from static functions
 	  and comment algorithm
 
-	Bugs #875 V IS ZERO AND, #880 error compiling abbreviated conditions
-	and #887 error compiling parenthesized relation
+	  Bugs #875 V IS ZERO AND, #880 error compiling abbreviated conditions
```

Comment:
this is actually a "topic note", those are recognizable because they don't include the leading spaces... (the others are fine now)